Desktop_Interface/isodriver.cpp

```diff
@@ -1148,25 +1127,28 @@ void isoDriver::slowTimerTick(){
     update_CH2 = true;
 }
 
-void isoDriver::setTopRange(double newTop){
-    topRange = newTop;
-    topRangeUpdated(topRange);
+void isoDriver::setTopRange(double newTop)
+{
+    // NOTE: Should this be clamped to 20?
```

Comment:
An assert here would be more appropriate.  If the newTop is > 20, something has gone horribly horribly wrong.